esp/templates/program/modules/programprintables/classes_list.html

```diff
@@ -56,8 +56,13 @@
  <td sorttable_customkey="{{ cls.id }}">{{ cls.emailcode }}</td>
  <td>{{ cls.title }}</td>
  <td>{{ cls.class_info|truncatewords_html:10 }}
+ {% if split_teachers %}
+ <td>{{ cls.split_teacher.name }}</td>
```

Comment:
I wonder if it would actually be clearer to show all teachers of the class -- so you sort it under each teacher, but show all of them.